classes/PaymentModule.php

```diff
@@ -202,6 +202,7 @@ public function addCheckboxCarrierRestrictionsForModule(array $shops = [])
      * @param bool $dont_touch_amount
      * @param string|bool $secure_key
      * @param Shop $shop
+     * @param string $order_ref order reference generated from outside when set this reference is taken
```

Comment:
```suggestion
     * @param string $order_reference if this parameter is not provided, a random order reference will be generated
```

Reply:
It sounds obvious for you 😄  but maybe we can be a little more user-oriented for people who would discover this function